maat-court-data-api/src/main/java/gov/uk/courtdata/billing/controller/ApplicantHistoryBillingController.java

```diff
@@ -0,0 +1,30 @@
+package gov.uk.courtdata.billing.controller;
+
+import gov.uk.courtdata.annotation.StandardApiResponseCodes;
+import gov.uk.courtdata.billing.service.ApplicantHistoryBillingService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@Slf4j
+@RestController
+@RequiredArgsConstructor
+@Tag(name = "Applicant History Billing Extract", description = "Rest API for extracting applicant history to send to billing teams")
+@RequestMapping("${api-endpoints.billing-domain}/applicant-history")
+public class ApplicantHistoryBillingController {
+
+    private final ApplicantHistoryBillingService applicantHistoryBillingService;
+
+    @GetMapping
+    @Operation(description = "Extract applicant history based on MAAT_REFS_TO_EXTRACT table")
```

Comment:
Optional suggestion, but I would maybe remove the bit about the `MAAT_REFS_TO_EXTRACT` table here as it's an implementation detail that we don't really need to expose.

Reply:
Yeh fair point I've removed the table mention from the lines of logging.

Reply:
Yeh this has been amended too.